src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -267,6 +272,21 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
     this.props.navigator.pop()
   }
 
+  presentErrorResult(error) {
+    console.error(error)
```

Comment:
I left this `console.error` to send the error to Sentry. In development this will show a red screen that may be surprising, but that's intentional.